src/Network/Ethereum/Web3/Types/Types.purs

```diff
@@ -258,7 +261,17 @@ instance showTransactionOptions :: Show (TransactionOptions u) where
   show = genericShow
 
 instance encodeTransactionOptions :: Encode (TransactionOptions u) where
-  encode = genericEncode (defaultOptions { unwrapSingleConstructors = true })
+  encode (TransactionOptions txOpts) =
+    let encodeMaybe :: forall a. Encode a => Maybe a -> Foreign
+        encodeMaybe = maybe undefined encode
+    in encode $ FO.fromFoldable [ Tuple "from" $ encodeMaybe txOpts.from
```

Comment:
Why do we need to do this now?

Reply:
because otherwise the non-specified options get encoded as `null` and the web3 tests fail.